.github/workflows/ci.yml

```diff
@@ -56,6 +72,8 @@ jobs:
         sudo ./ci/setup_cmake.sh
     - name: run tests
       run: ./ci/do_ci.sh cmake.c++20.test
+    #- name: run tests (enable stl)
```

Comment:
Log an issue to add this back?

Reply:
Yes, created now- #859

Reply:
And I already fixed it. Please merge the fix - #860